components/dashboard/conf/Caddyfile

```diff
@@ -12,29 +12,12 @@
 		path /static/* /favicon* /manifest.json
 	}
 
-    rewrite /static/bin/gitpod-local-companion-linux /static/bin/gitpod-local-companion-linux-amd64
```

Comment:
We should make sure that old links still work for backward compatibility by rewriting (not redirecting) them to ide proxy. It should be a part of how to test section.

It will also imply that we need to deploy before webapp team? Otherwise redirects won't work?